Initial event listing build

domsammut opened the pull request:
Utilise Preact for templating and add webpack hot reloading for development.

@jnowland potentially will need you to add some touches to it visually. Hopefully should be enough to get going.

neozenith commented:
![screen shot 2018-11-04 at 3 31 37 pm](https://user-images.githubusercontent.com/159722/47960022-f0f94280-e046-11e8-9763-b10634a84de8.png)

Looks great! I can see us pushing https://newwwie.com/#events a lot more in the future with this.

neozenith commented:
Not sure how you get the Meetup IDs but Rowan's meetup is missing:

https://www.meetup.com/Newcastle-Blockchain/

domsammut commented:
@neozenith I've added Newcastle Blockchain. I've also executed `npm run build` on the commits (as that appeared to be what @jnowland had done with the CSS).

The good thing about Preact (at least how I implemented it) as I didn't have to learn too much extra besides knowing React.
